XML/syntax_test_xml.xml

```diff
@@ -201,6 +222,26 @@
      >
 <!-- ^ text - punctuation - illegal -->
 
+     <NS:TAGNAME XMLNS:NS="URI">
+<!-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.tag -->
+<!--                            ^ - meta.tag -->
+<!-- ^ punctuation.definition.tag.begin -->
+<!--  ^^ entity.name.tag.namespace -->
+<!--    ^ punctuation.separator.namespace -->
+<!--     ^^^^^^^ entity.name.tag.localname -->
+<!--            ^ - entity -->
+<!--             ^^^^^ entity.other.attribute-name.namespace -->
```

Comment:
A browser and most other XML tools like Oxygen XML don't care about it at all, too. They all just highlight those constructs case-sensitive

What you complain about is not syntax highlighting but linting!

This is to be handled by SublimeLinter or Exalt.